src/lib.rs

```diff
@@ -181,9 +181,14 @@ pub use retry::Retryable;
 mod retry_with_context;
 pub use retry_with_context::RetryableWithContext;
 
+#[cfg(not(target_arch = "wasm32"))]
 mod blocking_retry;
+#[cfg(not(target_arch = "wasm32"))]
 pub use blocking_retry::BlockingRetry;
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
`BlockingRetry` can't work on wasm32?

Reply:
It does not make sense because it will block all execution since WASM is single-threaded. Therefore thread::sleep fails with "can't sleep". Since this is a design decision by Rust, I chose to comment out the blocking retry for wasm. See here for example: https://stackoverflow.com/q/65512520/233430

Reply:
Got it, thanks for the explanation.